packages/cli/openapi-parser/src/schema/convertSchemas.ts

```diff
@@ -641,7 +640,28 @@ export function convertSchemaObject(
             return maybeInjectDescriptionOrGroupName(convertedSchema, description, groupName);
         }
 
-        // otherwise convert as an object
+        // Now that we've handled the single-element allOf case, filter the
+        // allOfs down to just the objects.
```

Comment:
should we log which `allOfs` we are ignoring that are not objects? (maybe as a warn)

Reply:
Great call, I'll add that in